src/types/instantsearch.ts

```diff
@@ -22,3 +25,15 @@ export type NumericRefinement = {
 };
 
 export type Refinement = FacetRefinement | NumericRefinement;
+
+export type Client = AlgoliaSearchClient;
+
+export type Helper = AlgoliaSearchHelper;
+
+// We complete the original helper state with the custom SearchParameters
+// not yet typed in the original `algoliasearch-helper`.
+export type HelperState = AlgoliaSearchHelper['state'] & SearchParameters;
```

Comment:
Both `HelperState` and `SearchParameters ` describe the same object. Why do we have both?

Reply:
I kept only `SearchParameters` in the end.